src/styles/card.css

```diff
@@ -0,0 +1,84 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 166px;
+  padding: 0 16px 16px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background-color: #fff;
+  color: #060b35;
+  font-family: Roboto, sans-serif;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin: 32px 3px 0;
```

Comment:
2 points:
> let's set vertical paddings

- let's give this job to its parent container, you don't need top margin
- 3px — looks hardcoded, let's set precise width to image